modules/doc/content/application_development/build_system.md

```diff
@@ -208,3 +208,49 @@ when building MOOSE or a MOOSE-based application.
 
 - `framework/conf_var.mk.in` - This file is where you put your new expansion expressions. Typically these
   are in the form of "@VARAIBLE@".
+
+### Make Install
```

Comment:
Would be nice if we could set this up as a template so that application documentation could include it